airframe-di/.jvm/src/test/scala/wvlet/airframe/DesignSerializationTest.scala

```diff
@@ -13,11 +13,21 @@
  */
 package wvlet.airframe
 
-import java.io.{ByteArrayInputStream, ByteArrayOutputStream, ObjectInputStream, ObjectOutputStream}
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, InputStream, ObjectInputStream, ObjectOutputStream}
 
 import DesignTest._
 import wvlet.airspec.AirSpec
 
+class CustomClassLoader(in: InputStream) extends ObjectInputStream(in) {
```

Comment:
Note) This custom class loader fixes `java.lang.ClassCastException: cannot assign instance of scala.collection.generic.DefaultSerializationProxy to field wvlet.airframe.Design.binding of type scala.collection.immutable.Vector in instance of wvlet.airframe.Design`

Reply:
FYI: Serializing the design is necessary in td-spark for sending the design to remote Spark worker node  https://www.slideshare.net/taroleo/tdspark-internals-extending-spark-with-airframe-spark-meetup-tokyo-3-2020 

![image](https://user-images.githubusercontent.com/57538/166135228-3d2c519f-e5d7-4ba4-852d-1ad462c2abc6.png)